javascript/chronometer.js

```diff
@@ -1,34 +1,48 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = null;
   }
+  
 
   start(callback) {
-    // ... your code goes here
+    this.intervalID = setInterval(() => this.currentTime++, 1000)
```

Comment:
this.intervalId*
Be careful with casing on your code, right now you're not assigning the setInterval to your variable from the constructor